ui/actions/lighthouse/compliances.ts

```diff
@@ -0,0 +1,87 @@
+import { apiBaseUrl, getAuthHeaders, parseStringify } from "@/lib/helper";
+
+export const aiGetCompliancesOverview = async ({
```

Comment:
@Chan9390 could we keep the `app` folder just for routing purposes?

If this is an action (a call to the API), it should go inside the actions/lighthouse/compliances folder.

https://nextjs.org/docs/14/app/building-your-application